### ISSUE TITLE
Configure Transaction History to Display Pending by Configuration

### ISSUE THREAD

**Issue by Mike-E-angelo** [StarbeamOne/home#11]
Currently the user transaction history pulls in pending transactions by default, which is super slow. One person paging back and forth on the server takes up 30% of the DTU, which is really excessive.

- Update query to remove pending transactions by default
- Allow pending transactions to be included via switch

**Comment by Mike-E-angelo** [comment]
I've split these into two queries now for the user transaction history page.  Looks like DTU is 5-10% per user now.  Not perfect but better.
